appservice/src/getKuduClient.ts

```diff
@@ -3,14 +3,20 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { User } from 'azure-arm-website/lib/models';
+import { AppServicePlan, User } from 'azure-arm-website/lib/models';
 import { BasicAuthenticationCredentials } from 'ms-rest';
 import { addExtensionUserAgent } from 'vscode-azureextensionui';
 import KuduClient from 'vscode-azurekudu';
 import { ArgumentError } from './errors';
+import { localize } from './localize';
 import { SiteClient } from './SiteClient';
 
 export async function getKuduClient(client: SiteClient): Promise<KuduClient> {
+    if (!client.kuduHostName) {
+        const asp: AppServicePlan = await client.getAppServicePlan();
+        const notSupportedLinux: string = localize('notSupportedLinux', 'This operation is not supported by App Service Plans with kind "{0}" and sku tier "{1}"', client.kind, asp.sku.tier);
```

Comment:
Nit: Add a period at the end of the message and change "Plans" to "plans" (here's an example doc that uses lowercase "plans" all over: https://docs.microsoft.com/en-us/azure/app-service/azure-web-sites-web-hosting-plans-in-depth-overview)